lab3/src/lab3/calculator.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package lab3;
+
+import java.util.Scanner;
+
+/**
+ *
+ * @author MingJie
+ */
+public class calculator {
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+        double num1, num2, answer;
+        int operator;
+        print("Please enter a operator 1 for +, 2 for -, 3 for * or 4 for / ");

```

Comment:
:-1: From the lab specification:
> You should ask the user to input the following, each on a separate line: an operator (which can be +,
-, /, or *), an integer, and a second integer.